ui/next.config.js

```diff
@@ -24,7 +24,21 @@ module.exports = phase => ({
             value:
               phase === PHASE_DEVELOPMENT_SERVER
                 ? ''
-                : `default-src 'self'; img-src * 'self' data: https:;`,
+                : [
```

Comment:
I feel like this is worse since it requires a regex search/replace in order to add it properly as a header value.

```javascript
ContentSecurityPolicy.replace(/\s{2,}/g, ' ').trim()
```

IMO is much harder to understand than an array join.

I'm happy to change how the value is templated. I just don't think this is the right way.